pkg/v18/resource/clusterconfigmap/desired.go

```diff
@@ -20,8 +20,15 @@ func (r *StateGetter) GetDesiredState(ctx context.Context, obj interface{}) ([]*
 
 	configMapName := key.ClusterConfigMapName(clusterConfig)
 
+	// Calculating DNS IP from the IP range so we other operators could use it w/o processing it.
+	clusterDNSIP, err := key.DNSIP(r.clusterIPRange)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
 	values := map[string]string{
-		"baseDomain": key.DNSZone(clusterConfig),
+		"baseDomain":   key.DNSZone(clusterConfig),
+		"clusterDNSIP": clusterDNSIP,
```

Comment:
For my interest: why do we need that in chart-operator?

Reply:
We have a long history of this value; AFAIK we put this value to keep working on with current chart-operator even if CoreDNS have downtime (e.g. initial setup). 

Slack threads 10 months ago: https://gigantic.slack.com/archives/C02MS174K/p1537532882000100

Original PR: https://github.com/giantswarm/chart-operator/pull/139

Reply:
Thanks <3